main/webapp/modules/core/langs/translation-en.json

```diff
@@ -132,7 +132,7 @@
     "core-index-parser/discard-initial": "Discard initial",
     "core-index-parser/rows-data": "row(s) of data",
     "core-index-parser/load-at-most": "Load at most",
-    "core-index-parser/parse-cell": "Parse cell text into<br/>numbers, dates, …",
+    "core-index-parser/parse-cell": "Attempt to parse cell text into<br/>numbers.",
```

Comment:
None of the other labels on the screen are terminated by periods, so let's leave it off here too.

Also, the break `<br/>` looks a little awkward there with just a single word on the second line. Perhaps we can move it after the word "cell" to balance out the two lines.